exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -103,6 +109,8 @@ public Response getDrillbitStatus() {
   @GET
   @Path("/gracePeriod")
   @Produces(MediaType.APPLICATION_JSON)
+  @Operation(externalDocs = @ExternalDocumentation(description = "Apache Drill REST API documentation:", url = "  https://drill.apache.org/docs/stopping-drill/#:~:text=draining%2C%20and%20offline.%0A%20%20%20%20%20--,grace,-%3A%20A%20period%20in"))
```

Comment:
Please remove extra spaces after the `url` tag.  Here and elsewhere.  Also, is it necessary to URL encode the links, or will Drill figure that out by itself?

If it is not necessary, I'd recommend rendering them normally, w/o the URL encoding.

Reply:
The URL encoding came from copying the link to specific text on the page. Would you rather me remove the encoding and have the URLs point to the general page instead?